Formula/vde.rb

```diff
@@ -1,39 +1,26 @@
 class Vde < Formula
   desc "Ethernet compliant virtual network"
-  homepage "https://vde.sourceforge.io/"
-  url "https://downloads.sourceforge.net/project/vde/vde2/2.3.2/vde2-2.3.2.tar.gz"
-  sha256 "22df546a63dac88320d35d61b7833bbbcbef13529ad009c7ce3c5cb32250af93"
+  homepage "https://github.com/virtualsquare/vde-2"
+  url "https://github.com/virtualsquare/vde-2/archive/refs/tags/v2.3.3.tar.gz"
+  sha256 "a7d2cc4c3d0c0ffe6aff7eb0029212f2b098313029126dcd12dc542723972379"
   license "GPL-2.0"
```

Comment:
```suggestion
  license all_of: ["GPL-2.0-or-later", "LGPL-2.1-or-later"]
```
`GPL-2.0-or-later` for the project

`LGPL-2.1-or-later` for `libvdehist`